src/editor.js

```diff
@@ -15,7 +15,7 @@ const fireEvent = (node, type, detail = {}, options = {}) => {
   return event;
 };
 
-const OptionsArtwork = ['cover', 'full-cover', 'material', 'cover-fit', 'none'];
+const OptionsArtwork = ['cover', 'full-cover', 'material', 'full-cover-fit', 'none'];
```

Comment:
Let's sort it a little bit:

```suggestion
const OptionsArtwork = ['cover', 'full-cover', 'full-cover-fit', 'material', 'none'];
```